packages/nube-devtools/src/devtools/components/network-details-panel.tsx

```
@@ -0,0 +1,23 @@
+import { JsonViewer } from "./json-viewer";
+
+interface NetworkDetailsPanelProps {
+	selectedEvent: { id: string; data: any } | null;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)